src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -301,19 +307,17 @@ class PrisonerIepLevelReviewService(
               withDetails = true,
               useClientCredentials = true
             ).iepDetails
-          val iepLevelBeforeTransfer =
```

Comment:
there was a type mismatch in the calculation of `iepLevelBeforeTransfer`

`iepHistory.sortedBy(IepDetail::iepTime).lastOrNull { it.agencyId != prisonerInfo.agencyId }?.iepCode` is `String?`
`defaultLevel` is `IepLevel`
making `iepLevelBeforeTransfer` an `Any`